web_src/js/features/repo-diff.js

```diff
@@ -89,15 +92,25 @@ function initRepoDiffConversationForm() {
     const action = $(this).data('action');
     const url = $(this).data('update-url');
 
-    const data = await $.post(url, {_csrf: csrfToken, origin, action, comment_id});
+    const params = new URLSearchParams();
+    params.append('origin', origin);
+    params.append('action', action);
+    params.append('comment_id', comment_id);
```

Comment:
```suggestion
    const params = new URLSearchParams({origin, action, comment_id});
```

Contrary to `FormData`, this does accept a object.